build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+	id 'org.springframework.boot' version '2.6.4'
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+	id 'java'
+}
+
+group = 'com.miausoft'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '17'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.springframework.boot:spring-boot-starter-mail'
+	implementation 'org.springframework.boot:spring-boot-starter-security'
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.liquibase:liquibase-core'
+	compileOnly 'org.projectlombok:lombok'
+	runtimeOnly 'org.postgresql:postgresql'
+	annotationProcessor 'org.projectlombok:lombok'
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.springframework.security:spring-security-test'
```

Comment:
Not sure if it's related to Tests, but it seems so. I would like if you would remove everything that's is needed for tests as we wont write any of them

Reply:
doesn't bother me :)

Reply:
Well, these two lines doesn't bother me also but this class/folder does it.
![image](https://user-images.githubusercontent.com/60428415/159870647-6d694a0b-2952-43c9-ac36-a5fa159c19b1.png)

Reply:
Whatever you choose, resolve the conversation. I made a restriction that every conversation must be resolved before merge.